update_languages_android.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+source supported_languages
```

Comment:
I would put this after the check to see if you are in the correct directory and for sure after the `set -e` so it exits when it fails to import it.